lua/entities/gmod_wire_expression2/core/cl_http_cl.lua

```diff
@@ -0,0 +1,131 @@
+--[[
+
+	Clientside HTTP library
+	 - swadical 🍌
```

Comment:
@AbigailBuccaneer We have a bunch of things in wiremod that were committed back in SVN, and those commit logs have been lost. We have no way of contacting some of those people. I say having the author is fine, but not always helpful